Formula/cairo.rb

```diff
@@ -47,6 +54,7 @@ def install
       args << "--enable-xcb=no" << "--enable-xlib=no" << "--enable-xlib-xrender=no"
     end
 
+    system "./autogen.sh", *args if build.head?
```

Comment:
If this also expects the arguments to be passed directly to it it seems unlikely you want to repeat the process again below...